.gitmodules

```diff
@@ -28,3 +28,4 @@
 [submodule "vendor/libchromiumcontent"]
 	path = vendor/libchromiumcontent
 	url = https://github.com/electron/libchromiumcontent
+	branch = electron-2-0-x
```

Comment:
@jkleinsc Can it have any unexpected / undesirable effects?

Reply:
@alexeykuzmin this allows using `git submodule update --remote` to update to the latest version of the given branch. I can remote the change

Reply:
@miniak I don't mind having that thing there, but I just have never used it so was wondering if it could break something. Thanks!